src/components/Amount.tsx

```diff
@@ -23,7 +23,8 @@ interface AmountProps {
   className?: string
 }
 
-const Amount = ({ value, className }: AmountProps) =>
-  value ? <span className={className}>{abbreviateAmount(value)}א</span> : <span className={className}>- א</span>
+const Amount = ({ value, className }: AmountProps) => (
+  <span className={className}>{value !== undefined ? abbreviateAmount(value) : '-'} א</span>
```

Comment:
This adds a space between the value and the symbol. Is that OK?

![image](https://user-images.githubusercontent.com/1579899/148748705-768a87b3-7f66-491c-9e2b-72d44ae7c9a7.png)

Reply:
Yes, makes sense.